modules/ROOT/pages/migrate-ah.adoc

```diff
@@ -23,7 +23,7 @@ ssh_authorized_keys:
   - ssh-rsa ...
 ----
 
-Manually configure with a xref:ign-passwd.adoc[`passwd`] node in the FCC file:
+This can be manually translated into a xref:ign-passwd.adoc[`passwd`] node within the FCC file:
```

Comment:
Minor: Hmm, not sure if we should even refer to `ign-passwd.adoc`. We mention higher up already that the conversion should be to FCCs and link to that specification, so I think it'd be less confusing to just say something like

> The equivalent FCC for this would be:

?

Reply:
if you click on the generated link: [passwd](https://github.com/coreos/fedora-coreos-docs/blob/master/modules/ROOT/pages/ign-passwd.adoc) that document mostly describes the options and then gives example FCC configs. Is the link really not useful?

Reply:
Ohh I see, I thought the `ign-` nodes documented Ignition instead of FCC. But that's not the case (well, it does also document Ignition since those bits match, but the examples are of FCC).

Reply:
I wonder if those files should be named `fcc-<...>.adoc` instead?

Reply:
maybe :)